docs/scos/dev/guidelines/process-documentation-guidelines.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Process documentation guidelines
+description: @TODO
+last_updated: @TODO
+template: concept-topic-template
+related:
+- title: NFR guidelines
+  link: docs/scos/dev/guidelines/nfr-guidelines.html
+- title: Operatable feature guidelines
+  link: docs/scos/dev/guidelines/operatable-feature-guidelines.html
+- title: Monitorable process guidelines
+  link: docs/scos/dev/guidelines/monitorable-process-guidelines.html
+---
+
+The process documentation guidelines are in place to enhance the communication between cross-teams over all processes and to 
+ensure that they can operate and deploy their applications the best-possible way.
+
+## Deployment guidelines
+The following elements apply in the scope of deployment/rollback of each release candidate.
+
+### Environment variable assertion
+In huge-scaled teams, the human communication (and issue administration) may face many challenges - as a side-effect, some environment variable 
+or their expected value set may not reach the production environment as expected. To decrease the risk of a critical mistake in the line, we 
+offer to run a final validation on the production environment to match the expected setup change with the actual state. In order to understand 
+the scope of changes, we need the list of these handed over with the deployment standard service request.
```

Comment:
An example would be good here. Is that supposed to be a standard process on each deploy / on change..?

Reply:
I'm just trying to imagine I'm a customer. I see this recommendation here on the page, and think "Sounds awesome!" but, how am I supposed to do that, whom and when to contact and to send what and in which format.?

Reply:
I've added an example below